tests/functional/object/mcg/test_noobaa_use_new_internal_cert_after_rotation.py

```diff
@@ -0,0 +1,116 @@
+from datetime import datetime
+import logging
+
+from ocs_ci.ocs.constants import SECRET, NOOBAA_S3_SERVING_CERT
+from ocs_ci.ocs.exceptions import UnexpectedBehaviour
+from ocs_ci.framework.pytest_customization.marks import (
+    tier1,
+    polarion_id,
+    bugzilla,
+    skipif_ocs_version,
+    red_squad,
+    mcg,
+    config,
+)
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.pod import get_noobaa_endpoint_pods
+from ocs_ci.utility.retry import retry
+
+
+logger = logging.getLogger(__name__)
+
+
+@retry(UnexpectedBehaviour, tries=10, delay=3, backoff=1)
+def get_validity_time_of_certificate(noobaa_endpoint_pods, cmd, old_validity=None):
+    """
```

Comment:
This doc string needs to include the argument and return value details

Reply:
added